### PR TITLE
Change the defaulting for get_ENU_antpos

### PR DESCRIPTION
**Pull request by bhazelton**
To not remove the median antenna location

### CODE REVIEW

File: pyuvdata/uvdata.py [hunk 1]
```diff
@@ -2495,6 +2495,13 @@ def get_ENU_antpos(self, center=True, pick_data_ants=False):
         antpos : ndarray, antenna positions in TOPO frame and units of meters, shape=(Nants, 3)
         ants : ndarray, antenna numbers matching ordering of antpos, shape=(Nants,)
         """
+        if center is None:
+            center = False
+            warnings.warn('The default for the `center` keyword has changed. '
+                          'Previously it defaulted to True, using the mean '
```
Comment (plaplant) [inline pyuvdata/uvdata.py]: s/mean/median/

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file.
 - support for python3.7
 
 ### Changed
+- UVdata.get_ENU_antpos() now defaults to using the telescope_location as the center rather than the mean antenna position.
```
Comment (plaplant) [inline CHANGELOG.md]: I think we were using the median before, not the mean.

File: pyuvdata/uvdata.py (hunk 1 above)
Comment (bhazelton) [reply pyuvdata/uvdata.py]: thanks!

**Comment by bhazelton** [comment]
@plaplant I fixed the language. Thanks!

**Pull request merged**